liveHomeBackend/store/example/index.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+const config = require('../config/index')
+const db = require('../index')
+
+async function run () {
+  const configuration = {
+    database: config.db.name || 'livehome_db',
+    username: config.db.user || 'admin',
+    password: config.db.password || 'admin123',
+    host: config.db.host || 'localhost',
+    dialect: 'postgres',
+    plain: true
+  }
+
+  const { typeUser } = await db(configuration).catch(handleFatalError)
+
+  const typeUser5 = await typeUser.createOrUpdate({
+    id: '5',
+    name: 'luis',
+    description: 'el comprador',
+    createAt: new Date(),
+    updateAt: new Date()
+  }).catch(handleFatalError)
+
+  console.log('--userType4--')
+  console.log(typeUser5)
+
+  // const typeUser3 = await typeUser.createOrUpdate({
+  //   id: '3',
+  //   name: 'cliente',
+  //   description: 'el cliente',
+  //   createAt: new Date(),
+  //   updateAt: new Date(),
+  // }).catch(handleFatalError)
+  // console.log('--userType--')
+  // console.log(typeUser3)
```

Comment:
A pull request should not have code in comments

Reply:
ok. i will delete it . Thanks